todo_app/data/trello_static_data.py

```diff
@@ -0,0 +1,30 @@
+import os, requests
+
+myAPIKey = os.getenv('TRELLO_API_KEY')
+myToken = os.getenv('TRELLO_TOKEN')
+boardID = os.getenv('TRELLO_BOARD_ID')
+
+toDoListID = ""     
+doneListID = ""
+
+#Add error handling from trello API
+
+def get_trello_lists():
+    """
+    Fetches all lists and saved cards from Trello.
+
+    Returns:
+        list: All the items from Trello
+    """
+    payload = {'key': {myAPIKey}, 'token': {myToken}, 'cards': 'open', 'card_fields': 'name'}
+    trello_lists = requests.get(f'https://api.trello.com/1/boards/{boardID}/lists', params=payload).json()
+    return trello_lists
+
+trello_lists = get_trello_lists()
+
+for list in trello_lists:
+    list_name = list.get("name")
+    if list_name == "To Do":
+        toDoListID = list.get("id")
+    elif list_name == "Done":
+        doneListID = list.get("id")
```

Comment:
This works nicely here, but might catch you out in future exercises - primarily because this relies on the lists being loaded "at the right time" - i.e. after the variables are set, but before anyone uses the app - which can be fiddly particularly when testing. The suggestions below are definitely optional here, but might save you a headache in M3!

* The _easiest_ solution for the list ids is potentially a slightly cheesy one (but not a bad one!), but we could just put the list ids into environment variables, so that the `add_item` function can use something like: `'idList': {os.getenv("TRELLO_TODO_LIST_ID"}`
* For the others, I'd look at potentially making them functions that get called rather than variables that get set - that way they naturally only execute when they're actually needed:
```python
def myAPIKey():
    return os.getenv('TRELLO_API_KEY')


....


def get_trello_lists():
    """
    Fetches all lists and saved cards from Trello.

    Returns:
        list: All the items from Trello
    """
    payload = {'key': {myAPIKey()}, 'token': {myToken()}, 'cards': 'open', 'card_fields': 'name'}
    ...
```

Reply:
The fiddly reason this might matter is when we come to testing, we will want our tests to set some fake environment variables as part of the set up, but by default:
* The imports at the top of the test files will import some of your app code that we want to test
* They will in turn trigger their own imports, which will eventually include this file
* Then this file will try execute and talk to Trello - when it shouldn't as part of our testing!
* _Then_ we'll get to the part of the test that sets up the fake environment

We can fix that by very carefully specifying the order everything happens, but that typically leads to very fragile set ups that are fiddly to maintain

Reply:
I have refactored the code according to some of your suggestions and created another pull request. Please let me know if this is good enough to avoid any headaches in M3!